### ISSUE TITLE
Does the math add up?

### ISSUE THREAD

**Issue by stnbu** [stnbu/mass_gathering#8]
## Concerning commit 2d5f83a88f5087bb08b7bc2ae111bc56dfb28c46 ...

_**tl;dr -- "vector balls" are great but is all that math really right? I'm no newtonian. If you don't look at the code, does it seem intuitively right?**_

I have hosted a WASM build of "main" for the above commit hash [here](https://unintuitive.org/mass_gathering/builds/2d5f83a88f5087bb08b7bc2ae111bc56dfb28c46/), which should be enough to play along.

When a planet is "hot" (intersects your crosshairs) you will see three little balls floating in front of each planet's path of travel; in front of the "bow".

The [Orange ball is the Origin, the Maroon ball is the Momentum, the Fuchsia one is the Force](https://github.com/stnbu/mass_gathering/blob/2d5f83a88f5087bb08b7bc2ae111bc56dfb28c46/src/physics.rs#L329) (feel free to pick your own).

The origin flies at a _fixed_ distance from the planet and in the direction of its travel (The normal of the velocity times a constant).

Both the momentum and force balls are placed at the _tip_ of their respective vector (because graphics is hard). The vectors' frame of reference is the "global" frame of reference so, e.g. if the force on the planet is pointing straight up, that means its force direction is _your_ straight up. It could be pointing any direction in absolute terms. It doesn't matter. The force vector points in the direction the force is pulling on the planet. Simple.

The momentum vector is always pointing straight forward. This is because momentum is "velocity times mass". The _origin_ (orange) ball is riding out in front of the planet in the direction of its velocity vector, so it makes sense that the origin ball is between the momentum ball and the planet's center.

Weird thing number one: The force vector starts out at zero and grows over time. It takes some time to "emerge". Once it's fully out and extended, it seems to keep up with reality accurately. If a planet wanders near another planet, the force vector gradually starts to drift in that direction. Hard to see with so much noise.

On that topic, you can explore a much simpler system in [this example](https://github.com/stnbu/mass_gathering/blob/2d5f83a88f5087bb08b7bc2ae111bc56dfb28c46/examples/shooting_gallery.rs) at [this very fine url](https://unintuitive.org/mass_gathering/builds/2d5f83a88f5087bb08b7bc2ae111bc56dfb28c46/examples/shooting_gallery/).

[todo: implement shafts and more realistic tips]
